File: plugin/js/libraries/knex/v1_x/methods.js

function detect(input, callback) {
  let path = input.request.root;
  callback(null, {
    value: true,
    error: null,
  });
}

function isUsed(input, callback) {
  let path = input.request.root;
  callback(null, {
    value: true,
    error: null,
  });
}

function percentOfUsed(input, callback) {
  let path = input.request.root;
  callback(null, {
    error: null,
  });
}
module.exports = {
  detect: detect,
  isUsed: isUsed,
  percentOfUsed: percentOfUsed
};

File: plugin/python/libraries/numpy/v1_x/methods.py
from protos.plugin import library_pb2_grpc, common_pb2


class Methods(library_pb2_grpc.LibraryServiceServicer):
    def Detect(self, request, context):
        return common_pb2.ServiceOutputBool(value=True, error=None)

    def IsUsed(self, request, context):
        return common_pb2.ServiceOutputBool(value=True, error=None)

    def PercentOfUsed(self, request, context):
        return common_pb2.ServiceOutputFloat(value=78, error=None)
